module/mempool/consensus/exec_fork_suppressor.go

```diff
@@ -108,21 +108,6 @@ func (s *ExecForkSuppressor) Add(newSeal *flow.IncorporatedResultSeal) (bool, er
 	}
 	blockID := newSeal.Seal.BlockID
 
-	// STEP 2: enforce that newSeal's state transition does not conflict with other stored seals for the same block
```

Comment:
we don't enforce consistent state when adding anymore.

Reply:
better to mention that it's possible the seal is conflict with other seals for the same block, we will check that when builder finds seals to add to the block.